File: March/01-03-2024/main.py
number = int(input("Enter number : "))
answer = []

for i in range(number):
    if i <= 1:
        answer.append(1)
        continue
    answer.append(answer[-2] + answer[-1])

for i in range(len(answer)):
    print(f"{i+1})\t{answer[i]}")

File: March/01-03-2024/main.c
#include <stdio.h>

int main() {
    int number;
    printf("Enter number : ");
    scanf("%d", &number);

    int answer[number];

    for (int i = 0; i < number; i++) {
        if(i <= 1) {
            answer[i] = 1;
            continue;
        }

        answer[i] = answer[i - 2] + answer[i - 1];
    }

    for (int i = 0; i < number; i++) {
        printf("%d)\t%d\n",i+1, answer[i]);
    }

    return 0;
}